mkdocs.yml

```
@@ -114,66 +114,70 @@ extra:
+            - 'Services': 'admin/admin-ui/services-menu.md'
+            - 'SMTP': 'admin/admin-ui/smtp-menu.md'
+            - 'SCIM': 'admin/admin-ui/scim-menu.md'
+            - 'FIDO': 'admin/admin-ui/fido-menu.md'
+            - 'SAML': 'admin/admin-ui/saml-menu.md'
+            - 'Jans Lock': 'admin/admin-ui/jans-lock-menu.md'
+          - 'FAQ & Troubleshooting': 'admin/admin-ui/faq.md'
+      - 'Flex Recipes':
+          - 'admin/recipes/README.md'
+          - 'Testing Flex SAML SSO with sptest': 'admin/recipes/testing-flex-saml-sso-sptest.md'
+          - 'Add required claims to id-token': 'admin/recipes/add-claims-to-token.md'
+      -  'Migrating from Gluu4': 'admin/gluu-migration.md'
```

YAMLlint (1.38.0)

[error] too many spaces after hyphen

(hyphens)